internal/diagnosticwriter/diagnosticwriter.go

```diff
@@ -90,7 +90,7 @@ func writeCodeSnippet(writer io.Writer, sourceFile *ast.SourceFile, start int, l
 	lastLineOfFile, _ := scanner.GetLineAndCharacterOfPosition(sourceFile, len(sourceFile.Text))
 
 	hasMoreThanFiveLines := lastLine-firstLine >= 4
-	gutterWidth := len(strconv.Itoa(lastLineOfFile + 1 + len("")))
+	gutterWidth := len(strconv.Itoa(lastLine + 1))
```

Comment:
For reference, the original code is:

```ts
    const hasMoreThanFiveLines = (lastLine - firstLine) >= 4;
    let gutterWidth = (lastLine + 1 + "").length;
    if (hasMoreThanFiveLines) {
        gutterWidth = Math.max(ellipsis.length, gutterWidth);
    }
```

So your fix is definitely correct; the `+ ""` was a conversion.